src/dependencies.js

```diff
@@ -29,9 +29,13 @@
     onDocumentBody(initialize);
   }
 
-  if (!window.JSON) {
+  if (typeof(Pusher.runtime.getWindow().JSON)=== undefined) {
```

Comment:
We could move the fetching of `JSON` from the runtime instead.

Hmmm. Maybe even the dependencies should be loaded by the runtime.

Also - does this work? Here we're checking the value of a string against `undefined`. AFAIK that will always fail. Shouldn't it just be `Pusher.runtime.getWindow().JSON === undefined` ?